packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -15,6 +15,18 @@ if (typeof INFURA_KEY === 'undefined') {
 const MAINNET_INFURA_RPC_URL = `https://mainnet.infura.io/v3/${INFURA_KEY}`
 const GOERLI_INFURA_RPC_URL = `https://goerli.infura.io/v3/${INFURA_KEY}`
 
+export function getL2ChainIds(l1ChainId: number): ChainId[] {
+  if (l1ChainId === ChainId.Mainnet) {
+    return [ChainId.ArbitrumOne, ChainId.ArbitrumNova]
+  }
+
+  if (l1ChainId === ChainId.Goerli) {
+    return [ChainId.ArbitrumGoerli]
+  }
+
```

Comment:
Right now if we don't add this we are not able to use dropdowns in the local network. Doesn't matter much because we can only use one network, but could cause issues later when trying to write some tests.

```suggestion
  if (l1ChainId === ChainId.Local) {
    return [ChainId.ArbitrumLocal]
  }
```

Reply:
Good catch!